### DISCUSSION TITLE
第6节课笔记(10班)

### DISCUSSION THREAD

**Discussion by isLinXu** [InternLM/Tutorial#442] [笔记]
> 请在下方提交第 **6** 节课笔记链接，知乎/CSDN/飞书 等等都可。 笔记让知识记得更牢固💪

**Comment by lishuang70345** [comment]
https://blog.csdn.net/weixin_42748604/article/details/135722725

**Comment by admlrz** [comment]
https://blog.csdn.net/weixin_44581312/article/details/135758033

**Comment by hannah803** [comment]
https://hannahliu.framer.website/blog/internlm-training-lec6

**Comment by dearSocrates** [comment]
https://blog.csdn.net/rabbit9798/article/details/135773319?spm=1001.2014.3001.5501

**Comment by Tian-CASIA** [comment]
https://mp.csdn.net/mp_blog/creation/editor/135854159
